dataverse-persistence/src/test/resources/dbinit.sql

```diff
@@ -48,7 +48,7 @@ INSERT INTO guestbook (id, createtime, emailrequired, enabled, institutionrequir
 
 --- ROOT -> unreleased ---
 INSERT INTO dvobject (id, dtype, owner_id, previewimageavailable, protocol, authority, identifier, globalidcreatetime, identifierregistered, storageidentifier, releaseuser_id, publicationdate, creator_id, createdate, modificationtime, permissionmodificationtime, indextime, permissionindextime) 
-    VALUES (51, 'Dataverse', 1, false, NULL, NULL, NULL, NULL, false, NULL, 1, '2019-08-19 13:15:02.415', 1, '2019-08-19 13:14:48.434', '2019-08-19 13:15:02.415', '2019-08-19 13:14:48.799', '2019-09-19 14:37:07.268', '2019-09-19 14:37:07.486');
```

Comment:
I agree, but I only did that because it is supposed to be unreleased dataverse judging by description. But since it is determined by date this object was incorrect (null - unrealesed, any date - realesed).